t/View/TT.t

```diff
@@ -0,0 +1,77 @@
+use feature 'unicode_strings';
+use strict;
+use warnings;
+use Setup;
+
+my %ctx = test_init();
+
+require Hydra::Schema;
+require Hydra::Model::DB;
+
+use Test2::V0;
+
+require Hydra; # calls setup()
+
+
+my $db = Hydra::Model::DB->new;
+hydra_setup($db);
+
+require Hydra::View::TT;
+
+# The following lines are a cheap and hacky trick to get $c,
+# there is no other reason to call /.
+require Catalyst::Test;
+Catalyst::Test->import('Hydra');
+my($_, $c) = ctx_request('/');
+
+
+my $project = $db->resultset('Projects')->create({name => "tests", displayname => "", owner => "root"});
+my $jobset = createBaseJobset("example", "bogus.nix", $ctx{jobsdir});
+my $job = "myjob";
+
+
+is(
+    Hydra::View::TT::linkToProject(undef, $c, $project),
+    '<a href="http://localhost/project/tests">tests</a>',
+    "linkToProject"
+);
+is(
+    Hydra::View::TT::linkToJobset(undef, $c, $jobset),
+    '<a href="http://localhost/project/tests">tests</a>:'
+    . '<a href="http://localhost/jobset/tests/example">example</a>',
```

Comment:
If I understand the implementation correctly, it looks like the various `linkTo*` subs create the link with the colon leading -- I think it would make sense to also follow this convention here:

```suggestion
    '<a href="http://localhost/project/tests">tests</a>'
    . ':<a href="http://localhost/jobset/tests/example">example</a>',
```

And in all the following `is` tests.